persist/fs/files.go

```diff
@@ -245,17 +245,11 @@ func CommitLogFilesForTime(commitLogsDir string, t time.Time) ([]string, error)
 
 // CommitLogFilesBefore returns all the commit log files whose timestamps are earlier than a given time.
 func CommitLogFilesBefore(commitLogsDir string, t time.Time) ([]string, error) {
-	var multiErr xerrors.MultiError
-
 	commitLogs, err := CommitLogFiles(commitLogsDir)
 	if err != nil {
-		multiErr = multiErr.Add(err)
-	}
-	res, err := filesBefore(commitLogs, t)
-	if err != nil {
-		multiErr = multiErr.Add(err)
+		return nil, err
 	}
-	return res, multiErr.FinalError()
+	return filesBefore(commitLogs, t)
```

Comment:
Hm, it might also be worth fixing `filesBefore` to actually read the info header too.  Seems it has the same issue.

Reply:
This is resolved now 👍

Reply:
is this resolved in some other PR? I ended up not touching this package much at all

Reply:
Yeah I think we were thinking we should update this fn to use your new `ReadLogInfo(...)` method, do you mind doing that? 

We should also rename it to `commitLogFilesBefore(...)` or just inline it here, I checked and this is the only function that calls `filesBefore(...)`.

Reply:
Ok sure thing, maybe we can open an issue and come back to it?

Reply:
#391